pages/studentAssessmentInstanceExam/studentAssessmentInstanceExam.sql

```diff
@@ -28,3 +28,42 @@ WHERE
 WINDOW
     w AS (ORDER BY qo.row_order)
 ORDER BY qo.row_order;
+
+-- BLOCK get_group_info
```

Comment:
Is this query still used?

Reply:
I don't think so and neither is `leave_group`, so both removed in [c2c89fe](https://github.com/PrairieLearn/PrairieLearn/pull/5290/commits/c2c89feff699c155a0f9bf70f857676f1adf9bf1)